game.html

```diff
@@ -4,8 +4,17 @@
     <meta charset="utf-8">
     <title>Tic Tac Toe</title>
     <link rel="stylesheet" href="game.css">
+    <style>
+      body  {
+        background-image: url("paper.gif");
+        background-color: #cccccc;
+      }
+      </style>
   </head>
   <body>
+    <div id="main">
+      <p>Game starts by just Tap on box<br><br>First Player starts as<b>Player X</b><br>And<br>Second Player as <b>Player 0</b></p>
```

Comment:
You can remove the following instruction, we don't need them they are just decreasing UI quality